File: Python Code/Daily_Flash_Code_0/Week9/21Mar/53-DailyFlash_Solutions/21_March_Solution_Three/C/prog4.c
#include<stdio.h>

int main() {
	int num;
	for(int i = 0; i < 4; i++) {
		num = i * (i + 1);
		for(int j = 0; j < 7 - i; j++) {
			if(j < i) {
				printf("\t");
			} else {
				printf("%d\t",num);
				num++;
			}
		}
		printf("\n");
	}
}

File: Python Code/Daily_Flash_Code_0/Week9/21Mar/53-DailyFlash_Solutions/21_March_Solution_Three/Java/prog4.java
class Demo {
	public static void main(String[] args) {
		int num;
		for(int i = 0; i < 4; i++) {
			num = i * (i + 1);
			for(int j = 0; j < 7 - i; j++) {
				if(j < i) {
					System.out.print("\t");
				} else {
					System.out.print(num);
					System.out.print("\t");
					num++;
				}
			}
			System.out.println();
		}
	}
}
